stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -103,7 +103,7 @@ public final class _DataStorage {
     public var _length: Int
     public var _capacity: Int
     public var _needToZero: Bool
-    public var _deallocator: ((UnsafeMutableRawPointer, Int) -> Void)? = nil
+    public var _deallocator: ((UnsafeMutableRawPointer, Int) -> Void)?
```

Comment:
Like the ones in the standard library, I'd hesitate about changing these without asking someone from Foundation.